src/garage/torch/algos/vpg.py

```diff
@@ -7,6 +7,7 @@
 import torch
 import torch.nn.functional as F
 
+from garage import TorchTrajectoryBatch
```

Comment:
please make any updates to this algo their own PR. it will make QA and code review immensely easier.